pkg/dashboard/assets/css/main.css

```diff
@@ -529,3 +529,79 @@ UTILITIES
   white-space: nowrap;
   width: 1px;
 }
+
+.email-box {
+  background: #4c415f;
+  padding: 1.375rem 0.8125rem;
+  width: 100%;
+}
+
+.email-box__toggle-title {
+  color: #fff;
+  font-size: var(--step-4);
+  font-weight: 600;
+}
+
+.email-box__content p {
+  color: #fff;
+  padding: 0.75rem 0 1.5rem 0;
+}
+
+.email-box__email-label {
+  position: relative;
+}
+
+.email-box__email-label span {
+  color: #d3d3d3;
+  font-weight: 400;
+  left: 0;
+  left: 0.5rem;
+  position: absolute;
+  top: -8%;
+}
+
+.email-box__email-input {
+  background: #fff;
+  border-radius: 0.25rem;
+  border: 1px solid #d3d3d3;
+  margin-bottom: 1rem;
+  padding: 0.5rem;
+  width: 71%;
+}
+
+.email-box__email-input:focus-visible {
+  outline: thick solid #4d90fe;
+}
+
+.email-box__checkbox-label {
+  color: #fff;
+  display: block;
+  font-size: 1rem;
+}
+
+#email-box__checkbox {
+  accent-color: #445688;
```

Comment:
This color doesn't contrast well against the purple background; the [WebAIM: Contrast Checker](https://webaim.org/resources/contrastchecker/), calculates the contrast ratio as 1.31. This makes the focus style as well as the checked checkbox hard to see. Could we please increase the contrast here?

Reply:
I updated the code. Thank you for your feedback!

Reply:
Looks great! Thank you!